mobify/platform-scaffold
# [Integration manager] Remove duplicate login and register commands

This PR removes the checkout login & registration commands so we have just one command each for logging in and registering a user. 

 **JIRA**: n/a
 **Linked PRs**: n/a

## Changes
- Updated login and registration commands to accept arguments instead of a form values object
- Updated actions in the app that call the login & registration commands to pass correct data to the commands


## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- open the menu and click "sign in"
- try signing in with valid credentials
- log out
- navigate back to the sign in page
- try creating a user with valid credentials
- log out
- add an item to your cart and proceed to the checkout shipping page
- log in with valid credentials from that page
- log out
- add an item to your cart and proceed through the checkout to the order confirmation page 
** make sure you use an email address that doesn't have an account associated with it **
- enter a password & confirmation password on the page and create an account
